### ISSUE TITLE
the paper's 53.9mIou on SemanticKITTI validation set(08) 

### ISSUE THREAD

**Issue by fenfenglitech** [QingyongHu/RandLA-Net#201]
my training result is around 54.0mIou on the validation set(08),
and the paper's 53.9mIou is test from val(08) ?
but why my motorcyclist acc is 0 ?

**Comment by Talnex** [comment]
I tested 57% mIoU on the 08 sequence, and the results in the paper are in the online test (11 ~ 21 sequences)
